### PR TITLE
Seperate Card Links + Load more button improvement

### PR DESCRIPTION
**Pull request by Shaddamah**
I modify the card layout and seperate the link to 3 different links (Issue URL, Repository URL, Comments URL) and resolve the issue on #198 , sorry for taking your part @AbhinavReddy-Dev

image

I also create improvement for the **Load more** button to make it look disabled when data is still fetching

#### Before

image

#### After

image

### CODE REVIEW

**Comment by AbhinavReddy-Dev** [comment]
@Shaddamah Chill bro, there is one more change that you missed, make comments lighter than Issue title, make it grey. Hence, the colour hierarchy. 😁

**Review by damcclean — CHANGES_REQUESTED** [review]
Thanks for your pull request!

I've added a few comments for you to look over.

**Comment by Shaddamah** [comment]
Oh i'm sorry @damcclean, i use beautify to lint 1 block code but ending up messing all the file, i'll revert it back :)

File: src/js/app.js [hunk 1]
```diff
@@ -37,6 +37,7 @@ new Vue({
                 )
                 .then(response => response.json())
                 .then(issuesResponse => {
+                    console.log(issuesResponse);
```
[the remark this answers is not in the document]
Comment (Shaddamah) [reply src/js/app.js]: Yes, you're right, i'll remove it

**Comment by Shaddamah** [comment]
Fixed! please take a look @damcclean thank you.

**Comment by AbhinavReddy-Dev** [comment]
@Shaddamah Looks like the individual links (repo link and comments) are broken except for the Issue Title in the deploy/netlify preview.

**Comment by Shaddamah** [comment]
Oh you're right, thank you, fixed!

File: index.html [hunk 2]
```diff
@@ -34,64 +34,52 @@
             <div class="py-8">
                 <div class="container mx-auto flex items-center justify-between">
                     <a href="/">
-                        <img
-                            class="h-24"
-                            src="/img/hacktoberfest.svg"
-                            alt="Hacktoberfest Icon"
-                        >
+                        <img class="h-24" src="/img/hacktoberfest.svg" alt="Hacktoberfest Icon">
```
[the remark this answers is not in the document]
Comment (damcclean) [reply index.html]: Thanks for reverting those.

**Review by damcclean — APPROVED** [review]
Thanks again for your pull request!

**Pull request merged**